### ISSUE TITLE
Issue tracking README (projectDocs/issues/readme.md): Confusing paragraph about “translations”/locales

### ISSUE THREAD

**Issue by Chealer** [nvaccess/nvda#19949]
The third paragraph of [the file currently titled _Reporting issues or feature requests_](https://github.com/nvaccess/nvda/blob/e2a2868171d6d1bd006b2f018ea23bc7e6630e18/projectDocs/issues/readme.md) reads:
> Issues with translations should be reported to the [NVDA Translators list](https://groups.io/g/nvda-translations).

This was added following ticket #15627, but as explained there, that ticket misdiagnosed the problem. The actual issue with the ITS is now tracked in #19947, and even that does not technically prevent reporting locale bugs.

The current situation is confusing:
1. There is no explanation of what are “Issues with translations”.
2. Reporters of such issues are pointed to 2 places, one of which is not even an ITS.

I recommend to remove that paragraph, regardless of when #19947 is solved. I do not know whether the mailing list is notified when a l10n issue is reported, but it's certainly not automatic/immediate, since the reporting process does not differentiate l10n issues.

This also affects `projectDocs/dev/contributing.md`.

[🅭🄍](https://www.philippecloutier.com/Common+infrastructure+licensing#its)

**Comment by seanbudd** [comment]
Could you suggest some clarification on how the phrase "issues with translations" is unclear. I'm not sure what we could add to clarify it further.
Could you also clarify which 2 places that people are being pointed? I'm only aware of the translators list, which is where issues with translations should be reported.

**Comment by XLTechie** [comment]
> Could you suggest some clarification on how the phrase "issues with translations" is unclear. I'm not sure what we could add

I remember stumbling over that phrase at one point too, and wondering:

Does it refer to mistranslations? Missing translations? Missing languages? 
Technical problems in the translation mechanism? Bugs with NVDA's handling of 
translated strings?

I thought it was a safe bet that it was intended to mean the first three, and not likely the last two.

I.e. if it's a problem solvable by code, then it shouldn't go to the mailing 
list. If it's a problem solvable by humans making translations, it should go 
to the list.

But perhaps that distinction isn't clear to everyone? Or perhaps that's a 
conclusion only a developer is likely to reach?

**Comment by Chealer** [comment]
@seanbudd wrote:
> Could you suggest some clarification on how the phrase "issues with translations" is unclear. I'm not sure what we could add to clarify it further.

Luke has done it better than I could have. But if I may, if we are talking about existing strings, before you can even tell if that string has a translation issue, you have to know whether it's a translation. It's non-trivial to tell which languages a message was first written in, and which strings were translated from these originals.

Cheaper development was not the only reason for the design of GitHub's ITS; reporters used to deal with Bugzilla and the like were sick of having to set a priority, a component and fill other cryptic fields (or [being forced to select a version when the issue affects none](https://github.com/nvaccess/nvda/issues/19947)) just to report an issue, so GitHub KISSed. I'm not saying it gets it right, but every additional complication required from reporters heightens the barrier. GitHub has attracted unsophisticated trackers, many of whom have no idea what differentiates i18n from l10n. Even asking them to subscribe to and deal with a mailing list is an early barrier.

Should the narrator pronouncing English as an utterly conspicuous Frenchwoman be reported to that mailing list? And in fact, even if the paragraph used a simple designation like "Issues in text part of NVDA's interface or documentation", it's not _always_ obvious whether an issue is in documentation or behavior. We want a mismatch between these tracked even if reporters don't have enough analytical skills and/or product knowledge to tell where the defect is.

Locale issues are rarely major, yet tracking and fixing them early is valuable, if only because issues in one locale often spread to other languages. The earlier the report, the less work needs to be redone.

> Could you also clarify which 2 places that people are being pointed?

The first paragraph sends them here (the ITS). But the third says "translation issues" should be reported to the [mailing] list. This is pretty confusing. As far as I know this project (which is far from far), there is no integration between the ITS and that list, but many projects automatically notify a mailing list when reporting, and there are some email addresses which automatically create a ticket, so new contribusers can be quite confused about whether they should manually report to both places, and which one otherwise.

If there is no integration, I understand it's best to inform maintainers of locales, but this can be phrased as a request:
> If you file a ticket, please also notify [the NVDA Translators mailing list](https://groups.io/g/nvda-translations).

The above is just a suggestion and should be validated; it's not clear whether reporting a typo deserves notifying hundreds of subscribers.

**Comment by seanbudd** [comment]
Thanks @XLTechie - for clarifying, I've opened #19964 to close this
